src/tool/cppxlang/strings/base_string_operators.h

```diff
@@ -26,9 +26,9 @@ namespace winrt
         return left == std::wstring_view(right);
     }
 
-    bool operator==(hstring const& left, nullptr_t) = delete;
+    bool operator==(hstring const& left, std::nullptr_t) = delete;
```

Comment:
Any particular reason?

Reply:
Standards conformance, clang rejects it otherwise.